examples/main/README.md

```diff
@@ -341,6 +341,15 @@ These options help improve the performance and memory usage of the LLaMA models.
 
 For information about 4-bit quantization, which can significantly improve performance and reduce memory usage, please refer to llama.cpp's primary [README](../../README.md#prepare-and-quantize).
 
+## LoRA (Low-Rank Adaptation) adapters
+
+-   `--lora FNAME`: Optional path to a LoRA adapter to use with scaling of 1.0. Can be mixed with `--lora-scaled` and can be repeated to use multiple adapters.
+-   `--lora-scaled FNAME`: Optional path to a LoRA adapter with user-defined scaling. Can be mixed with `--lora` and can repeated to use multiple adapters.
+
+You can add LoRA adapters using `--lora` or `--lora-scaled`. For example: `--lora my_adapter_1.gguf --lora my_adapter_2.gguf ...` or `--lora-scaled lora_task_A.gguf 0.5 --lora-scaled lora_task_B.gguf 0.5`.
+
+LoRA adapters should be in GGUF format. To convert from Hugging Face format use the `convert-lora-to-gguf.py` script. GGUF files include base model information, so the `--lora-base` flag has been removed. LoRA adapters are loaded separately and applied during inference - they are not merged with the main model - so mmap model loading is fully supported.
```

Comment:
I'd say that the `--lora-base` is removed because lora is loaded separately (not merged). The `--lora-base` was used because users may want to use more precise quants for lora-merging.

So I'd appreciate if we can change some wordings here.

Reply:
Thank you @ngxson. I have changed the wording - hopefully it looks better now.